tailwind.config.js

```diff
@@ -110,6 +110,9 @@ module.exports = {
         '.special-increment-list-counter': {
           'counter-increment': 'list-counter',
         },
+        '.special-content-space': {
```

Comment:
@Entkenntnis any Idea why this is not working?

Reply:
`content: '" "'` is doing the trick

Reply:
Thank you!